src/components/Modal/README.md

```diff
@@ -5,102 +5,305 @@ Use the modal component to enter the user into a new _mode_.
+
+```html
+<template>
+	<m-modal>
+		Modal content
+
+		<!-- modalApi is provided by the injected the modalApi key below -->
+		<button @click="modalApi.close()">
+			Close modal
+		</button>
+	</m-modal>
+</template>
+
+<script>
+import { MModal, modalApi } from '@square/maker/components/Modal';
+
+export default {
+	components: {
+		MModal,
+	},
+
+	inject: {
+		modalApi,
+	},
+};
+</script>
+```
+
+To open a modal programmatically, import `modalApi` and _inject_ it into your component to access the Modal Layer API. In the function you want to open the modal from (eg. a click-event handler), invoke `this.modalApi.open()` with a function that returns the modal instance. This function receives [`createElement`](https://vuejs.org/v2/guide/render-function.html#createElement-Arguments) (aliased to `h`) as an argument to instantiate the modal component with, but it's recommended to use the [Vue JSX Babel plugin](https://vuejs.org/v2/guide/render-function.html#createElement-Arguments) instead for better readability.
+
+```html
+<template>
+	<button @click="openModal">
+		Open modal
+	</button>
+</template>
+
 <script>
 import { modalApi } from '@square/maker/components/Modal';
-import ModalDemo from 'doc/ModalDemo.vue';
+import MyModal from './MyModal.vue';
 
 export default {
+	// Bind the modal API
 	inject: {
 		modalApi,
 	},
 
 	methods: {
 		openModal() {
-			this.modalApi.open(() => <ModalDemo />);
+			// Use the Vue JSX Babel plugin to instantiate the modal component declaratively
+			this.modalApi.open(() => (
+				<MyModal
+					someProp="prop value"
+					vOn:someEvent={this.eventHandler}
+				/>
+			));
+
+			// Alternatively, use the createElement API
+			this.modal.open(h => h(MyModal, {
+				props: {
+					someProp: 'prop value',
+				},
+				on: {
+					someEvent: this.eventHandler,
+				},
+			}));
 		},
 	},
 };
 </script>
 ```
+## Examples
+
+### Stacking Modals
 
-_ModalDemo.vue_
 
 ```vue
 <template>
-	<m-modal>
-		<div class="cover-photo">
-			<m-image
-				src="https://picsum.photos/900"
-			/>
-		</div>
+	<div>
+		<m-button
+			size="small"
+			@click="openModal"
+		>
+			Open modal
+		</m-button>
+		<m-modal-layer />
+	</div>
+</template>
+
+<script>
+import { MButton } from '@square/maker/components/Button';
+import { MModalLayer } from '@square/maker/components/Modal';
+import StackedDemoFirstModal from 'doc/StackedDemoFirstModal.vue';
+
+export default {
+	name: 'StackedDemoSetup',
+
+	components: {
+		MModalLayer,
+		MButton,
+	},
+
+	mixins: [
+		MModalLayer.apiMixin,
+	],
+
+	methods: {
+		openModal() {
+			this.modalApi.open(() => <StackedDemoFirstModal />);
+		},
+	},
+};
+</script>
+```
+
+_StackedDemoFirstModal.vue_
+
+```vue
+<template>
+	<m-modal disable-pan-down-close>
```

Comment:
I thought we were going to disable the behavior by default and enable through a prop? Does this need more discussion / alignment before doing so?

Reply:
at this point it'd be better off as a docs example, we can do it in a follow-up PR